### PR TITLE
Updates routing logic

### PR DESCRIPTION
**Pull request by TimRobinson1**
Quick update to the routing logic - if the user is currently signed in, root takes you to THE WALL.  Else, it takes you to the signup page.

### CODE REVIEW

**Review by adamjohnsnow — APPROVED** [review]
Looks good to me (might want to clean up the home_helper_spec)

File: spec/controllers/home_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe HomeController, type: :controller do
+
+end
```
Comment (neoeno) [inline spec/controllers/home_controller_spec.rb]: I'd suggest deleting this file

File: app/assets/stylesheets/home.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Home controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (neoeno) [inline app/assets/stylesheets/home.scss]: I'd suggest deleting this file

File: app/assets/javascripts/home.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```
Comment (neoeno) [inline app/assets/javascripts/home.js]: I'd suggest deleting this file

**Review by neoeno — CHANGES_REQUESTED** [review]
There's some docs in here that might have saved you a little work — https://github.com/thoughtbot/clearance ("Clearance also provides routing constraints that can be used to control access at the routing layer:") — but I'll leave that up to you whether you like it or not.

Let's get these unused files out of the way and then we can merge

**Comment by TimRobinson1** [comment]
We added some changes, including using the built in Clearance constraints (thank you, Kay!) and removed the un-needed files.  Users who are not signed in now have no access to posts and must first sign_up/sign_in

**Comment by neoeno** [comment]
Great! I'd delete home_controller, home_helper, and home_controller_spec. I think I'm right in saying they're no longer required?

**Comment by TimRobinson1** [comment]
Right, I've removed those, you're right about them being no longer required.

File: app/helpers/home_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
+end
```
Comment (neoeno) [inline app/helpers/home_helper.rb]: One more!

**Review by neoeno — CHANGES_REQUESTED** [review]

**Review by neoeno — APPROVED** [review]

**Comment by neoeno** [comment]
Great work!

**Comment by neoeno** [comment]
(Go ahead and merge when you're ready, btw!)

**Review by neoeno — APPROVED** [review]
Still looks good!

**Pull request merged**